docs/conf.py

```diff
@@ -49,17 +49,17 @@
 
 # General information about the project.
 project = u'Tezos Reward Distributor'
-copyright = u'2019, habanoz'
```

Comment:
the date is normally stating since when a copyright is claimed, changing it would mean that the copyright from 2020 onwards and not before

Reply:
oh, ok. No worries.